packages/components/src/components/Warning/Warning.tsx

```diff
@@ -103,7 +103,6 @@ const Wrapper = styled.div<WrapperParams>`
     display: flex;
     ${typography.hint}
     gap: ${spacingsPx.sm};
-    justify-content: ${({ $withIcon }) => !$withIcon && 'center'};
```

Comment:
I really dont like that `withIcon` changes the alignment. It shall be independent

Reply:
(Alignment from the icon presence)